### ISSUE TITLE
Can't push over http

### ISSUE THREAD

**Issue by rnveach** [gitlabhq/gitlabhq#6926]
Whenever I try to push over HTTP, I get the following error:

```
$ git push -u origin master
Username for 'http://192.168.1.201': rnveach
Password for 'http://rnveach@192.168.1.201': 
Counting objects: 4, done.
Compressing objects: 100% (2/2), done.
Writing objects: 100% (4/4), 286 bytes | 0 bytes/s, done.
Total 4 (delta 0), reused 0 (delta 0)
remote: GitLab: You are not allowed to access master!
remote: error: hook declined to update refs/heads/master
To http://192.168.1.201/gitlab/astrowars/jaws.git
 ! [remote rejected] master -> master (hook declined)
error: failed to push some refs to 'http://192.168.1.201/gitlab/astrowars/jaws.git'
```

I followed the commands exactly on the project page to create the bare repo on an external pc. I can clone and pull the repo just fine. The user is added under the group and the project as owner and master. I tried with just the user under the group not the project, but got the same result.

```
$ sudo -u git -H bundle exec rake gitlab:env:info RAILS_ENV=production

System information
System:		Ubuntu 12.04
Current User:	git
Using RVM:	no
Ruby Version:	2.0.0p353
Gem Version:	2.0.14
Bundler Version:1.6.2
Rake Version:	10.1.1
Sidekiq Version:2.17.0

GitLab information
Version:	6.8.1
Revision:	3197990
Directory:	/home/git/gitlab
DB Adapter:	mysql2
URL:		http://<server>/gitlab
HTTP Clone URL:	http://<server>/gitlab/some-project.git
SSH Clone URL:	git@<server>:some-project.git
Using LDAP:	no
Using Omniauth:	no

GitLab Shell
Version:	1.9.3
Repositories:	/home/git/repositories/
Hooks:		/home/git/gitlab-shell/hooks/
Git:		/usr/local/bin/git
```

/home/git/repositories/ exists and /home/git isn't a symbolic link.

**Comment by Razer6** [comment]
Likely to be fixed in GitLab 8.0 with the new gitlab-http-server

**Comment by planetwebsolution** [comment]
Nothing worked in my case. I don't know why but at last I was stuck. Finally I removed by myself. Please take a look on my answer https://gitlab.com/gitlab-com/support-forum/issues/40. I hope it may help some one.
